packages/core/parcel-bundler/src/assets/CSSAsset.js

```diff
@@ -125,6 +148,48 @@ class CSSAsset extends Asset {
         'module.exports = ' + JSON.stringify(this.cssModules, null, 2) + ';';
     }
 
+    let map;
+    if (this.options.sourceMaps) {
+      if (this.sourceMap) {
+        if (this.sourceMap instanceof SourceMap) {
+          map = this.sourceMap;
+        } else {
+          map =
```

Comment:
This line isn't necessary, `addMap` should already be able to handle a sourcemap both in string and object form